resource/systemd/unit/preparer.go

```diff
@@ -25,42 +25,33 @@ import (
 
 // Preparer for Systemd Unit
 type Preparer struct {
-	// The name of this unit as in "foo.service"
+	// The name of this unit as in `foo.service`
+	// (either just file names or full absolute paths if the unit files are
+	// residing outside the usual unit search paths
 	Name string `hcl:"name"`
 
-	// Active determines whether this unit should be active or inactive
+	// Active determines whether this unit should be in an active or inactive state
 	Active string `hcl:"active"`
 
-	/* State is the UnitFileState of the unit.
-	UnitFileState encodes the install state of the unit file of FragmentPath.
-	It currently knows the following states: enabled, enabled-runtime, linked,
-	linked-runtime, masked, masked-runtime, static, disabled, invalid. enabled
-	indicates that a unit file is permanently enabled. enable-runtime indicates
-	the unit file is only temporarily enabled, and will no longer be enabled
-	after a reboot (that means, it is enabled via /run symlinks, rather than /etc).
-	linked indicates that a unit is linked into /etc permanently, linked indicates
-	that a unit is linked into /run temporarily (until the next reboot). masked
-	indicates that the unit file is masked permanently, masked-runtime indicates
-	that it is only temporarily masked in /run, until the next reboot.
-	static indicates that the unit is statically enabled, i.e. always enabled and
-	doesn't need to be enabled explicitly. invalid indicates that it could not
-	be determined whether the unit file is enabled.
+	/* state is the `UnitFileState` of the unit.
+	`UnitFileState` encodes the install state of the unit file of FragmentPath.
+	It currently knows the following states: `enabled`, `enabled-runtime`, `linked`,
+	`linked-runtime`, `masked`, `masked-runtime`, `static`, `bad`, `disabled`, `invalid`.
+
+	Of these states only  `enabled`, `enabled-runtime`, `linked`,
+	`linked-runtime`, `masked`, `masked-runtime`, `static`, `disabled`, and
+	`invalid` are permitted in the preparer
+
+	See https://godoc.org/github.com/coreos/go-systemd/dbus for more info
```

Comment:
But we should also be explaining how to use our implementation here as well instead of punting the reader off to another page. Could you summarize?